client/js/components/heading.js

```diff
@@ -13,7 +13,6 @@ const styles = {
 		'font-weight': 'bold',
 		'letter-spacing': '3.73px',
 		'line-height': '24px',
-		'justify-self': 'start',
```

Comment:
Is this supposed to be here? My guess is you probably need to merge the `polish-general-info` into this branch.